enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -243,6 +243,55 @@ const questionId = 1; // This is the question ID you want to embed
 
 ```
 
+### Embedding a static dashboard
+
+After the SDK is configured, you can embed your dashboard using the `StaticDashboard` component.
+
+The API follows the same configuration as that of public dashboard embeddings, which you can read about (here)[https://www.metabase.com/docs/latest/questions/sharing/public-links#public-embed-parameters]
+
+
+#### Parameters
+
+- **dashboardId**: `DashboardId` (required) – The ID of the dashboard to embed.
+- **parameterQueryParams**: `Query` (required) – Query parameters for the dashboard.
```

Comment:
Have we exposed these internal types to be user-facing? If `Query` and `DashboardId` is a simple type alias, might be worth it to indicate what types these actually are.

Reply:
Yep, that's fair. Here's what they are under the hood:

*   `Query` is just a key/value string - this is the `Record<[key: string]: T | T[] | null | undefined;>` type that you've seen before
*   `DashboardId` is `number | string` 

I'll modify the comments to reflect this

Reply:
Ok, I modified it - the ID should actually always be a number in this case (since strings are used in the public and static embedding contexts), so I'll just put `number`